portal/v2/src/ClusterDetail.tsx

```diff
@@ -281,6 +300,30 @@ export function ClusterDetailPanel(props: {
           <Stack.Item>
             <div className={headerStyles.titleText}>{props.currentCluster?.name}</div>
             <div className={headerStyles.subtitleText}>Cluster</div>
+            <TooltipHost content={`Copy Resource ID`}>
+              <IconButton
+                iconProps={{ iconName: "Copy" }}
+                aria-label="Copy Resource ID"
+                onClick={_onCopyResourceID.bind({},props.currentCluster)}
+              />
+            </TooltipHost>
+            <TooltipHost content={`Prometheus`}>
+              <IconButton
+                iconProps={{ iconName: "BIDashboard" }}
+                aria-label="Prometheus"
+                href={
+                  props.currentCluster?.resourceId + (+(version) >= 4.11 ? `/prometheus` : `/prometheus/graph`)
+                }
+              />
+            </TooltipHost>
+            <TooltipHost content={`SSH`}>
+              <IconButton
+                iconProps={{ iconName: "CommandPrompt" }}
+                aria-label="SSH"
+                onClick={() => _onSSHClick(props.currentCluster)}
+              />
+            </TooltipHost>
```

Comment:
You can create a separate component for this and use that in here and in the `ClusterListComponent`.

Reply:
New "ToolIcons" component created and used in both Cluster Listing and Detail Page